modules/ossm-rn-new-features.adoc

```diff
@@ -15,6 +15,51 @@ Module included in the following assemblies:
 
 This release adds improvements related to the following components and concepts.
 
+== New features {SMProductName} version 2.5.0
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs), contains bug fixes, and is supported on {product-title} 4.11 and later versions.
+
+[IMPORTANT]
+====
+{product-title} is designed for FIPS. When running on RHEL or RHEL CoreOS booted in FIPS mode, OpenShift Container Platform core components use the RHEL cryptographic libraries submitted to NIST for FIPS Validation on only the x86_64, ppc64le, and s390X architectures. For more information about the NIST validation program, see link:https://csrc.nist.gov/Projects/cryptographic-module-validation-program/validated-modules[Cryptographic Module Validation Program]. For the latest NIST status for the individual versions of the RHEL cryptographic libraries submitted for validation, see link:https://access.redhat.com/articles/2918071#fips-140-2-and-fips-140-3-2[Compliance Activities and Government Standards].”
+====
+
+=== Component versions included in {SMProductName} version 2.5.0
+
+|===
+|Component |Version
+
+|Istio
+|1.18
+
+|Envoy Proxy
+|1.24.12
+
+|Jaeger
+|1.47.0
+
+|Kiali
+|1.73.11
+|===
+
+=== OpenShift Service Mesh Console plugin
+
+This release introduces a generally available version of the OpenShift {SMProductShortName} Console (OSSMC) plugin. 
+
+The OSSMC plugin is an extension to the OpenShift Console that provides visibility into your Service Mesh. With the OSSMC plugin installed, a new Service Mesh menu option on the left-hand navigation of the web console, as well as new Service Mesh tabs that enhance existing Workloads and Service console pages. 
+
+The features of the OSSMC plugin are very similar to those of the standalone Kiali Console. The OSSMC plugin does not replace the Kiali Console, and after installing the OSSMC plugin, you can still access the standalone Kiali Console. 
+
+=== Univeral Base Image 9
```

Comment:
*Universal Base Image 9

Reply:
I see what you did you there, mi amigo! The fix has been submitted.